include/deal.II/hp/fe_collection.h

```diff
@@ -505,6 +505,26 @@ namespace hp
     void
     set_default_hierarchy();
 
+    /**
+     * Returns a sequence of FE indices that corresponds to the registered
+     * hierarchy in ascending order, i.e., FE indices are sorted from lowest to
+     * highest level.
+     *
+     * Multiple sequences of FE indices are possible with a single custom
+     * hierarchy, which can be registered with set_hierarchy(). This function
+     * will return the sequence that contains the user-provided index
+     * @p fe_index. The default hierarchy consists of only one sequence.
+     *
+     * This function can be used, for example, to verify that your provided
+     * hierarchy covers all elements in the desired order.
+     *
+     * Only one sequence of FE indices exists if the size of the returned
+     * container equals the number of elements of this object, i.e.,
+     * FECollection::size().
+     */
+    std::deque<unsigned int>
```

Comment:
That's uncommon :-) We generally use `std::vector` unless there is a good reason because popping is required from both front and back.